src/transactions/transferFeedUtils.ts

```diff
@@ -400,3 +427,52 @@ export function isTransferTransaction(
 ): tx is TransferItemFragment {
   return (tx as TransferItemFragment).address !== undefined
 }
+
+// Note: This hook is tested from src/transactions/feed/TransferFeedItem.test.ts
+function useFiatConnectTransferDisplayInfo({ amount, transactionHash }: TokenTransfer) {
+  const { t } = useTranslation()
+  const tokenInfo = useTokenInfo(amount.tokenAddress)
+  const fcTransferDetails = useSelector(getCachedFiatConnectTransferSelector(transactionHash))
+  const cachedFiatAccountUses = useSelector(cachedFiatAccountUsesSelector)
+  const localCurrency = useSelector(getLocalCurrencyCode)
+  const account = useMemo(
+    () =>
+      cachedFiatAccountUses.find(
+        (account) => account.fiatAccountId === fcTransferDetails?.fiatAccountId
+      ),
+    [cachedFiatAccountUses, fcTransferDetails]
+  )
+  if (!account || !fcTransferDetails) {
+    return
+  }
+
+  const receive = Number(fcTransferDetails.quote.fiatAmount)
+  const total = Number(amount.value)
+  const exchangeRate = String(receive / total)
+  const sign = fcTransferDetails.quote.transferType === TransferType.TransferOut ? -1 : 1
+  const localAmount: LocalAmount = {
+    value: new BigNumber(fcTransferDetails.quote.fiatAmount).multipliedBy(sign),
+    currencyCode: convertToLocalCurrency(fcTransferDetails.quote.fiatType) ?? localCurrency,
+    exchangeRate,
+  }
+
+  let subtitle: string
+  switch (account.fiatAccountType) {
+    case FiatAccountType.BankAccount:
+      subtitle = t('feedItemFcTransferBankAccount')
+      break
+    case FiatAccountType.MobileMoney:
+      subtitle = t('feedItemFcTransferMobileMoney')
+      break
+    default:
+      // should never happen
```

Comment:
this actually seems relatively likely to happen. In any case, is it really worth erroring out? Seems like a debug log and returning null would be sufficient

Reply:
Added the debug log